### ISSUE TITLE
A11y_SQLAzureDataStudio_SchemaCompare_Comparison_ScreenReader: Status message not announced for the data that is updated on the screen.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9401]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yWCAG2.1;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#MAS4.1.3;#Benchmark; #A11y_SQLAzureDataStudio;#A11ySev2;#NVDA;#JAWS;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS reference: MAS4.1.3
Screen Reader: NVDA(2019.3.1),JAWS

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Compare menu from Schema compare Menu bar & hit enter.
12. Check if status message is announced for all the updated content on the screen.

**Actual:**
The data that is updated on the bottom side of the comparison screen is not announced by screen reader.

**Issue observed with NVDA and JAWS.**

**Expected:**
All the data that gets updated on the screen should be announced by the screen reader.

**User Impact:**
Screen reader users will not receive all the information that is updated on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**Attachment link reference:**
[9401_A11y_SQLAzureDataStudio_Extension_SchemaCompare_Comparison_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291370/9401_A11y_SQLAzureDataStudio_Extension_SchemaCompare_Comparison_ScreenReader.zip)

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by Harika1702** [comment]
As per the discussion with our Accessibility SME this is an expected behavior as it is very hard to the screen reader to announce the content which is being changed while navigating in a table. So, closing the bug.
GitHub Tags: #Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
